lib/Frontend/ModuleInterfaceSupport.cpp

```diff
@@ -130,7 +131,11 @@ static void printToolVersionAndFlagsComment(raw_ostream &out,
         ignorableFlags, [&out](StringRef str) { out << str; },
         [&out] { out << " "; });
 
-    out << " -interface-compiler-version " << version::getCompilerVersion();
+    auto currentVersion = version::Version::getCurrentLanguageVersion();
+    // Prints only major + minor to maintain compatibility with older
+    // compilers.
+    out << " -interface-compiler-version " << currentVersion[0] << '.'
```

Comment:
Should we have a `FIXME:` tracking eventually printing the complete version?

Reply:
I think major + minor should be enough since we cannot actually check at more granularity anyway.